changelogs/fragments/58734-correct-machinectl-become-plugin.yaml

```diff
@@ -0,0 +1,3 @@
+bugfixes:
+  - machinectl become plugin - call proper get_option method
+  - machinectl become plugin - correct configuration option
```

Comment:
I think you can replace both lines with something like `  - machinectl become plugin - fix bugs which prevented plugin from working` :)